src/components/multilineInput/styles/SplitMultilineInput.css

```diff
@@ -0,0 +1,22 @@
+.inputBlock {
+  margin-top: 0;
+}
+
+.inputLine {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+
+  padding-bottom: 1em;
+}
+
+.input {
+  flex: auto;
+  margin-right: 2em;
+  margin-bottom: 1em;
+}
+
+.input:last-child {
+  margin-right: 0;
+  padding-right: 2em;
+}
```

Comment:
Does it add a style tag to the header?

I wonder how the `:last-child` work

Reply:
https://github.com/css-modules/css-modules/blob/master/docs/pseudo-class-selectors.md

All pseudo classes and html tag work as long as they have another class you can import.

Reply:
There is a new style tag, but I'm not sure if it is always there. As in, I don't know if it only gets added when the code is being executed.

Reply:
There is also https://github.com/gajus/react-css-modules but I do not see any mention of it not always adding the style tag either.